pom.xml

```diff
@@ -27,6 +27,553 @@
     ZK framework is licensed under LGPL and the license can be found at lgpl-3.0.txt
 
 -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I wonder what is going on with these changes (such a large diff). Usually adding a new pattern requires adding only one line to this file.